auslib/test/admin/views/test_releases.py

```diff
@@ -870,3 +870,16 @@ def testSettings(self):
         ret = self._get('/releases/settings')
         self.assertStatusCode(ret, 200)
         self.assertEqual(json.loads(ret.data), json.loads(data['blob']))
+
+
+class TestSingleColumn_JSON(ViewTest, JSONTestMixin):
+
+    def testGetReleasesSingleColumn(self):
+        expected_product = ["a", "c", "b", "d"]
+        expected = dict(count=4, product=expected_product)
+        ret = self._get("/releases/columns/product")
+        self.assertEquals(json.loads(ret.data), expected)
```

Comment:
It looks like this test is going to fail sometimes due to unpredictable ordering of the list. You've got "expected_product" sorted already, so you probably need to sort ret["product"] before comparing. You might be better off with two asserts, one for ret["count"], and one for ret["product"]. Other than that, this patch looks good!